packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2081,20 +2092,15 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
         [CISCO_DEVICE_URL]: this.webex.internal.device.url,
         [SPARK_USER_AGENT]: CALLING_USER_AGENT,
       },
-      body: {
-        device: {
-          deviceId: this.deviceId,
-          correlationId: this.correlationId,
-        },
-        callee: {
-          type: this.destination.type,
-          address: this.destination.address,
-        },
-        localMedia: {
-          roap: roapMessage,
-          mediaId: uuid(),
-        },
-      },
+      body: this.destination
```

Comment:
Only add a callee if it is not a guest calling. 
Note that in this case we will have a blank destination (for flows where user purposefully does not enter destination, it will not reach this flow itself).